index.html

```diff
@@ -9,6 +9,8 @@
 
 <body>
     <h1 style="color:red">This is my welcome page</h1>
```

Comment:
Make the header bold and add a '!' after 'This is my welcome page'.

Reply:
Ok, will do.